# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/nginx/more-autoindex-preallocation.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- nginx-1.29.2/src/http/modules/ngx_http_autoindex_module.c.vanilla	2025-10-15 11:54:59.220585216 +0100
+++ nginx-1.29.2/src/http/modules/ngx_http_autoindex_module.c	2025-10-15 11:55:36.203773851 +0100
@@ -50,9 +50,9 @@
 #define NGX_HTTP_AUTOINDEX_JSONP        2
 #define NGX_HTTP_AUTOINDEX_XML          3
 
-#define NGX_HTTP_AUTOINDEX_PREALLOCATE  50
+#define NGX_HTTP_AUTOINDEX_PREALLOCATE  100
 
-#define NGX_HTTP_AUTOINDEX_NAME_LEN     50
+#define NGX_HTTP_AUTOINDEX_NAME_LEN     100
 
 
 static ngx_buf_t *ngx_http_autoindex_html(ngx_http_request_t *r,
